mysql/create-aion.sql

```diff
@@ -360,12 +360,12 @@ create table account_stats
 
 CREATE TABLE `circulating_supply` (
   `timestamp` datetime NOT NULL,
-  `supply` decimal(16,1) DEFAULT NULL,
+  `supply` decimal(32,18) NOT NULL,
```

Comment:
This will give us a maximum of 32-18 digits of non-scale precision (left of the period), I think we won't be hitting this limit anytime soon :) so LGTM.

Reply:
great!